README.md

```diff
@@ -19,7 +19,7 @@ Note that all of these commands have been tested and used on Mac/unix machines,
 $ nvm use
```

Comment:
@MandyMeindersma: Thanks for fixing the `npm run install` command. The `nvm use` command alone does nothing on my machine even though I have nvm installed on it. I think the set up instructions need to be updated a bit still. Can be done in a separate PR.

Reply:
<img width="536" alt="image" src="https://github.com/devedmonton/DES-Website/assets/17459171/580b6dc1-8f59-4107-a794-a1b98466970e">


Oh weird. Mine worked yesterday?

Reply:
I'm using Windows. I believe `nvm use` expects a node version when we use it in Windows.